js/templates/feedback.html

```diff
@@ -9,7 +9,7 @@
     <form id="feedback-form" class="container">
       <fieldset>
         <legend class="feedback__title">Help us improve betaFEC</legend>
-        <p class="t-sans">Don't include sensitive information like your name, contact information or social security number.</p>
+        <p class="t-sans">Don't include sensitive information like your name, contact information or Social Security number.</p>
```

Comment:
FEC doesn't use the serial comma.

However, 18F style (and I) both prefer it. :)

On Wed, Nov 4, 2015 at 3:22 PM, Joshua Carp <notifications@github.com>
wrote:

> In js/templates/feedback.html
> <https://github.com/18F/fec-style/pull/172#discussion_r43934014>:
>
> > @@ -9,7 +9,7 @@
> >      <form id="feedback-form" class="container">
> >        <fieldset>
> >          <legend class="feedback__title">Help us improve betaFEC</legend>
> > -        <p class="t-sans">Don't include sensitive information like your name, contact information or social security number.</p>
> > +        <p class="t-sans">Don't include sensitive information like your name, contact information or Social Security number.</p>
>
> What's our position on the serial comma? Should this be "name, contact,
> information, or Social Security number"?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/18F/fec-style/pull/172/files#r43934014>.
>